pull-dependencies

```diff
@@ -11,13 +11,24 @@
 $version = '2.0'
 $isPublic = true
 
+# BEGIN_HIDE
```

Comment:
Dependencies are tricky since we are hosting them on the NLP cluster. Currently `./pull-dependencies` acts differently depending on the `$isPublic` variable. My original plan is to remove the HIDE part when the code is merged into master.